moveit2.repos

```diff
@@ -1,8 +1,4 @@
 repositories:
-  moveit2:
-    type: git
-    url: https://github.com/ros-planning/moveit2
-    version: main
```

Comment:
I see your point of only listing upstream stuff here. But, removing the main source will probably invalidate our build-from-source instructions, which essentially reads like this: "Fetch moveit2.repos and build." They should be adapted then.

Reply:
I'll prepare a PR for that.  This being here breaks the CI system because it creates a vcs error when it tries to import this file and moveit2 is already there.

Reply:
https://github.com/ros-planning/moveit.ros.org/pull/577

Reply:
What about `vcs import --skip-existing`?

Reply:
This caused the docker images to not have moveit2 source code, I think we should revert and apply @henningkayser suggestion, @rhaschke @tylerjw what do you think .?

Reply:
As @tylerjw already pointed out, `vcs import` is performed by `industrial_ci`. What about adding the MoveIt source explicitly in the docker image?

Reply:
Because the docker build is running from the repo you should be able to use the docker COPY command to get the moveit repo in docker.  See this line in the nav2 docker file: https://github.com/ros-planning/navigation2/blob/e13459441dd4895450b2f2073f73bf340823fcfe/Dockerfile#L26